config_gen.py

```diff
@@ -175,7 +175,10 @@ def fake_build(project_dir, c_build_log_path, cxx_build_log_path, verbose, make_
     else:
         # Preserve HOME, since Cmake needs it to find some packages and it's
         # normally there anyway. See #26.
-        env = dict(map(lambda x: (x, os.environ[x]), ["HOME"]))
+        if sys.version_info[0] == 2:
+            env = dict(map(lambda x: (x, os.environ[x]), ["HOME"]))
+        if sys.version_info[0] == 3:
+            env = dict([(x, os.environ[x]) for x in ["HOME"]])
```

Comment:
I'm not sure why you changed this line - the original seems to work in Python 3 - but if you're going to rewrite it, please use a dictionary comprehension. It will be clearer and should work in both versions.